perf_dashboard/static/js/pattern_common_func.js

```diff
@@ -42,15 +42,16 @@ window.generateChart = function(chartID, modesData) {
       itemclick: toggleDataSeries
     },
     data: [
-        {
-        type: "line",
-        showInLegend: true,
-        name: "baseline",
-        markerType: "square",
-        xValueFormatString: "DD MMM, YYYY",
-        color: "rgba(236, 66, 53, 1)",
-        dataPoints: modesData[0]
-      }, {
+      //   {
+      //   type: "line",
```

Comment:
should this be removed?

Reply:
yes, I commented this out, since baseline format is a little bit different from other modes. Which blocks the trending pattern lines to show up correctly. As we will move forward to using Mako for that part, later on all of these code will be deprecated.

Reply:
If its removed we can just remove it, no need to keep commented lines. If we need them later git has them